packages/optimizer/lib/transformers/ReorderHeadTransformer.js

```diff
@@ -60,12 +60,12 @@ class HeadNodes {
     appendChild(head, this._metaCharset);
     appendChild(head, this._styleAmpRuntime);
     appendChild(head, this._linkStyleAmpRuntime);
```

Comment:
`styleAmpRuntime` & `linkStyleAmpRuntime` order needs to be swapped as well.

See: https://github.com/ampproject/amppackager/blob/581d266a49d13384679697031cdd2a921f454d0c/transformer/transformers/reorderhead.go#L94-L99

Reply:
the order is irrelevant for these two as they are mutually exclusive (amp-runtime is the inlined version of v0.css).

Reply:
Changed the order for consistency and to avoid future confusion.